### PR TITLE
Creating files to consult from currencyLayer

### PR DESCRIPTION
**Pull request by alex-romanov**
Created currencylayer file an mock data, 
updated config.js file

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -22,8 +22,16 @@ const config = convict({
...
       default: '',
       env: 'FIXER_TOKEN',
-    },
+    }
   },
+  currencyLayer: {
+    token: {
+      doc: 'Access token',
+      format: String,
+      default: 'b7e9097ad7389732c12a86ecc47baeb0',
```
Comment (garciadiazjaime) [inline src/config.js]: better to not hardcode token,
`default: ''`
it can be added to your environment variables`

File: src/main.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import fixerEtl from './sources/fixer';
+import clEtl from './sources/currencyLayer'
 
-fixerEtl()
```
Comment (garciadiazjaime) [inline src/main.js]: to keep both ETL's, let's do:
```
Promise.all([fixerEtl(), clEtl()])
  .then(console.log);
```

File: src/sources/currencyLayer.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (garciadiazjaime) [inline src/sources/currencyLayer.js]: remove empty line

File: .babelrc [hunk 4]
```diff
@@ -1,5 +1,14 @@
...
+    [
+      "transform-runtime",
+      {
+        "polyfill": false,
+        "regenerator": true
+      }
+    ]
   ]
-}
+}
```
Comment (garciadiazjaime) [inline .babelrc]: let's add an `empty line` (your IDE should have a plugin to automatically do this whenever a file is saved)

File: src/sources/currencyLayer.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+import config from '../config'
+import extract from '../util/extract.js'
+import load from '../util/load'
+
+const props = {
+  isProduction: config.get('env') === 'production',
+  file: './stubs/currencyLayer.json',
+  currencyLayerUrl:`${config.get('currencyLayer.token')}0&currencies=USD,AUD,CAD,PLN,MXN&format=1`,
```
Comment (garciadiazjaime) [inline src/sources/currencyLayer.js]: it seems you are missing the `url`, you just have `token&...` it should be something like `http://api.com?token=...`

File: src/sources/currencyLayer.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    return accumulator
+  }, [])
+
+  return wantedRates
+}
+
+const main = async () => {
+  return extract(props)
+    .then(response => transform(props, response))
+    //.then(currencies => load(props, currencies))
```
Comment (garciadiazjaime) [inline src/sources/currencyLayer.js]: let's uncomment this line .. it's ok if it fails

File: stubs/currencyLayer.json [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  "timestamp": 1525322404,
+  "source": "USD",
+  "quotes": {
+    "USDUSD": 1,
+    "USDAUD": 1.328799,
+    "USDCAD": 1.28465,
+    "USDPLN": 3.567027,
+    "USDMXN": 19.036598
+  }
+}
```
Comment (garciadiazjaime) [inline stubs/currencyLayer.json]: empty line

File: src/sources/currencyLayer.js (hunk 6 above)
Comment (alex-romanov) [reply src/sources/currencyLayer.js]: done

File: src/main.js (hunk 2 above)
Comment (alex-romanov) [reply src/main.js]: done

File: src/sources/currencyLayer.js (hunk 3 above)
Comment (alex-romanov) [reply src/sources/currencyLayer.js]: done

File: src/sources/currencyLayer.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+ const props = {
+   isProduction: config.get('env') === 'production',
+   file: './stubs/currencyLayer.json',
+   sourceUrl:`http://apilayer.net/api/live?access_key=${config.get('currencyLayer.token')}0&currencies=USD,AUD,CAD,PLN,MXN&format=1`,
+   apiUrl: config.get('api.url'),
+   currencies: ['USDPLN', 'USDCAD', 'USDMXN']
+ }
+
+ const transform = (props, response) => {
```
Comment (garciadiazjaime) [inline src/sources/currencyLayer.js]: hay que sacar el `transform` de aqui .. similar a:
https://github.com/garciadiazjaime/etl-currency/blob/master/src/transformer/fixer.js
y agregar sus tests
https://github.com/garciadiazjaime/etl-currency/blob/master/tests/transformer/fixer.test.js

**Comment by garciadiazjaime** [comment]
good job dude!

**Pull request merged**